### DISCUSSION TITLE
can i run this on unity ?

### DISCUSSION THREAD

**Discussion by elmasveloz** [vladmandic/human#400] [Q&A] [answered]
can i run this on unity , if so what would i need >?

**Comment by vladmandic** [comment]
Most likely, yes. But I cannot give you advice how since I'm not actively using Unity.

**Reply by elmasveloz** [reply]
can it run mobile?
does it run without the need of internet?
on what engine did you make this ?
i have a real good idea for this.

**Reply by vladmandic** [reply, marked as answer]
mobile: yes.
without internet: yes
engine: tensorflow/js

**Reply by elmasveloz** [reply]
do you have an apk for this that i can install on my mobile for testing?
can you make it use the worldview camara not the frontal cam?

**Reply by vladmandic** [reply]
apk? no, i don not package an apk. this is a web app.
front vs back camera is configurable.

**Reply by elmasveloz** [reply]
thanks , ill try to complie this in unity . i would like to share some great ideas for this. , thanks for the responses
